envs/monkey_zoo/blackbox/config_templates/powershell.py

```diff
@@ -10,9 +10,10 @@ class PowerShell(ConfigTemplate):
     config_values.update(
         {
             "basic.exploiters.exploiter_classes": ["PowerShellExploiter"],
-            "basic_network.scope.subnet_scan_list": ["10.2.3.45", "10.2.3.46"],
+            "basic_network.scope.subnet_scan_list": ["10.2.3.45", "10.2.4.46"],
             "basic.credentials.exploit_password_list": ["Passw0rd!"],
-            "basic_network.scope.depth": 2,
+            "basic_network.scope.depth": 3,
+            "internal.general.keep_tunnel_open_time": 180,
```

Comment:
We shouldn't need to keep the tunnel open for this long. Do we know why it takes so long for the second machine to call back to the first?

Reply:
As the exploiter is fast, the first machine finishes before the  second communicates back. I think I can lower the time of the open tunnel.